README.md

```diff
@@ -302,6 +302,9 @@ This program, by default, scans 5000 ports at a time (5000 per second).
 
 This may cause damage to a server, or may make it incredibly obvious you are scanning the server.
 
+Servers as well as individual services listening on ports can block your ip completely after a huge amount of simultaneous connections/tries. 
```

Comment:
Servers as well as individual services can block your IP completely after a huge amount of simultaneous connections/tries, Possibly causing you to see false positives. False positives are open ports at the initial scanning process later found closed when you want to do further scanning, or connect to it.

Reply:
Changed.